<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Data</class>
 <widget class="QDialog" name="Data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="zatwierdz">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="Tytul">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ravie</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wpisz daty wyjazdu i przyjazdu</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>1030</width>
     <height>45</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="Dzienn1">
        <item>
         <widget class="QLabel" name="Dzien">
          <property name="text">
           <string>Dzień</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="DzienWpisz">
          <property name="styleSheet">
           <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Miech1">
        <item>
         <widget class="QLabel" name="Miesiac">
          <property name="text">
           <string>Miesiąc</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="MiesiacWpisz">
          <property name="styleSheet">
           <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Rok1">
        <item>
         <widget class="QLabel" name="Rok">
          <property name="text">
           <string>Rok</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="RokWpisz">
          <property name="styleSheet">
           <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>160</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="Dzienn1_2">
        <item>
         <widget class="QLabel" name="Dzien_2">
          <property name="text">
           <string>Dzień</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="DzienWpisz_2">
          <property name="styleSheet">
           <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Miech1_2">
        <item>
         <widget class="QLabel" name="Miesiac_2">
          <property name="text">
           <string>Miesiąc</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="MiesiacWpisz_2">
          <property name="styleSheet">
           <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Rok1_2">
        <item>
         <widget class="QLabel" name="Rok_2">
          <property name="text">
           <string>Rok</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="RokWpisz_2">
          <property name="styleSheet">
           <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
